packages/react/src/components/SignIn/SignIn.tsx

```diff
@@ -82,6 +82,7 @@ const SignIn: FC<SignInProps> = (props: SignInProps): ReactElement => {
   const {config} = useConfig();
 
   const authContext: AuthContext | undefined = useContext(AsgardeoContext);
+  const {setAuthResponse, setIsComponentLoading} = authContext;
```

Comment:
It's always better to access the context via a hook.
Lets track an issue to wrap the `AsgardeoContext` with a custom hook (ex: `useAsgardeo`)

Reply:
Tracked with https://github.com/asgardeo/web-ui-sdks/issues/49